### PR TITLE
JS: add alt description to image examples

### PR DESCRIPTION
**Pull request by estelle**
All images should have alternative text, including ones added with javascript.

### CODE REVIEW

File: files/en-us/web/html/cors_enabled_image/index.md [hunk 1]
```diff
@@ -70,15 +70,17 @@ The code that starts the download (say, when the user clicks a "Download" button
...
+We're using a hard-coded URL and associdated descriptive text here (`imageURL`), but that could easily come from anywhere. To begin downloading the image, we create a new {{domxref("HTMLImageElement")}} object by using the {{domxref("HTMLImageElement.Image", "Image()")}} constructor. The image is then configured to allow cross-origin downloading by setting its `crossOrigin` attribute to `"Anonymous"` (that is, allow non-authenticated downloading of the image cross-origin). An event listener is added for the {{domxref("Window/load_event", "load")}} event being fired on the image element, which means the image data has been received. Alternative text is added to the image; while `<canvas>` does not support the `alt` attribute, the value can be used to set an `aria-label` or the cavas's inner content.
```
Comment (Josh-Cena) [inline files/en-us/web/html/cors_enabled_image/index.md:83]:
```suggestion
We're using a hard-coded URL (`imageURL`) and associated descriptive text (`imageDescription`) here, but that could easily come from anywhere. To begin downloading the image, we create a new {{domxref("HTMLImageElement")}} object by using the {{domxref("HTMLImageElement.Image", "Image()")}} constructor. The image is then configured to allow cross-origin downloading by setting its `crossOrigin` attribute to `"Anonymous"` (that is, allow non-authenticated downloading of the image cross-origin). An event listener is added for the {{domxref("Window/load_event", "load")}} event being fired on the image element, which means the image data has been received. Alternative text is added to the image; while `<canvas>` does not support the `alt` attribute, the value can be used to set an `aria-label` or the cavas's inner content.
```

Also, the information about canvas seems a bit orthogonal for this particular example, but I'm fine if you want to put it here.

File: files/en-us/web/html/cors_enabled_image/index.md [hunk 2]
```diff
@@ -93,6 +95,7 @@ function imageReceived() {
 
   canvas.width = downloadedImg.width;
   canvas.height = downloadedImg.height;
+  canvas.innerText = downloadedImg.src;
```
Comment (Josh-Cena) [inline files/en-us/web/html/cors_enabled_image/index.md:98]: Is this accessible? I'm not sure. Maybe one line comment would help. Or maybe this is a typo for `downloadedImg.alt`?

File: files/en-us/web/javascript/reference/global_objects/promise/any/index.md [hunk 3]
```diff
@@ -115,6 +115,7 @@ Promise.any([coffee, tea])
     const objectURL = URL.createObjectURL(value);
     const image = document.createElement("img");
     image.src = objectURL;
+    image.alt = value;
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/promise/any/index.md:118]: This doesn't look accessible... Maybe it has to be a hardcoded string, or I can come back to see how the example can be refactored when I get to my computer.

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Great addition, but there are some I'm confused about.

File: files/en-us/web/html/cors_enabled_image/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/web/html/cors_enabled_image/index.md:98]: You are correct! Good catch. thank you.

Comment (estelle) [inline files/en-us/web/html/cors_enabled_image/index.md:98]:
```suggestion
  canvas.innerText = downloadedImg.alt;
```

File: files/en-us/web/javascript/reference/global_objects/promise/any/index.md (hunk 3 above)
Comment (estelle) [reply files/en-us/web/javascript/reference/global_objects/promise/any/index.md:118]: I thought `value` would be coffee or tea. Is there a way to pass the const name?

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/promise/any/index.md:118]: `value` is the result of `fetchAndDecode()`, which is a binary blob. Let's substitute this with:

```js
async function fetchAndDecode(url, description) {
  const res = fetch(url);
  if (!res.ok) {
    throw new Error(`HTTP error! status: ${response.status}`);
  }
  const data = await response.blob();
  return [data, description];
}

const coffee = fetchAndDecode("coffee.jpg", "Coffee");
const tea = fetchAndDecode("tea.jpg", "Tea");
Promise.any([coffee, tea])
  .then(([blob, description]) => {
    const objectURL = URL.createObjectURL(blob);
    const image = document.createElement("img");
    image.src = objectURL;
    image.alt = description;
    document.body.appendChild(image);
  })
```

Comment (estelle) [reply files/en-us/web/javascript/reference/global_objects/promise/any/index.md]: Thank you!

File: files/en-us/web/html/cors_enabled_image/index.md [hunk 4]
```diff
@@ -70,15 +70,17 @@ The code that starts the download (say, when the user clicks a "Download" button
...
+We're using a hard-coded URL (`imageURL`) and associated descriptive text (`imageDescription`) here, but that could easily come from anywhere. To begin downloading the image, we create a new {{domxref("HTMLImageElement")}} object by using the {{domxref("HTMLImageElement.Image", "Image()")}} constructor. The image is then configured to allow cross-origin downloading by setting its `crossOrigin` attribute to `"Anonymous"` (that is, allow non-authenticated downloading of the image cross-origin). An event listener is added for the {{domxref("Window/load_event", "load")}} event being fired on the image element, which means the image data has been received. Alternative text is added to the image; while `<canvas>` does not support the `alt` attribute, the value can be used to set an `aria-label` or the cavas's inner content.
```
Comment (Josh-Cena) [inline files/en-us/web/html/cors_enabled_image/index.md:83]:
```suggestion
We're using a hard-coded URL (`imageURL`) and associated descriptive text (`imageDescription`) here, but that could easily come from anywhere. To begin downloading the image, we create a new {{domxref("HTMLImageElement")}} object by using the {{domxref("HTMLImageElement.Image", "Image()")}} constructor. The image is then configured to allow cross-origin downloading by setting its `crossOrigin` attribute to `"Anonymous"` (that is, allow non-authenticated downloading of the image cross-origin). An event listener is added for the {{domxref("Window/load_event", "load")}} event being fired on the image element, which means the image data has been received. Alternative text is added to the image; while `<canvas>` does not support the `alt` attribute, the value can be used to set an `aria-label` or the canvas's inner content.
```

**Review by Josh-Cena — APPROVED** [review]
Looks great to me. Thanks!
